File: Data Structures & Algorithms/top-k-elements-in-list/submission-1.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        myMap = {}
        for i in nums : 
            if i in myMap :
                myMap[i] +=1
            myMap[i] = 1
        arr = myMap.values()
        heapq.heapify(arr)
        j = 0
        while j < k:
            arrFin = list(heapq.heappop(arr))
            k+=1
        return arrFin

File: Data Structures & Algorithms/two-integer-sum/submission-1.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int [] tab = new int[];
        for (int i = 0 ; i < nums.length(); i++) {
            for (int j = i+1 ; j < nums.length ; j++) {
                if (nums[i] + nums[j] = target) {
                    tab.add(i);
                    tab.add(j);
                }
            }
        }
        return tab;
    }
}
